receiver/redisreceiver/metadata.yaml

```diff
@@ -200,14 +201,14 @@ metrics:
   redis.memory.fragmentation_ratio:
     enabled: true
     description: Ratio between used_memory_rss and used_memory
-    unit: ""
+    unit: "{ratio}"
```

Comment:
Not sure I agree with this one. "ratio" describes the value in a way, but it's not a unit of measurement.

Reply:
Agreed, ratios are `unitless` -- perhaps I should put "{no unit: ratio}" ?

Reply:
👍 I believe "1" is the default unit, but it's ok to just omit it.

Reply:
I'm working on validation of the `metadata.yaml` ([issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23424)), and one of the checks that I currently have makes sure that the unit is not empty because it is a required field. This is an edge case, as the metric is unitless.  
I've made a commit to put the unit to 1 for `redis.memory.fragmentation_ratio` instead of leaving it empty -- let me know if this is OK ?

Reply:
I think 1 is the correct unit for ratio